lib/racecar/config.rb

```diff
@@ -153,6 +153,9 @@ class Config < KingKonf::Config
     desc "Whether to boot Rails when starting the consumer"
     boolean :without_rails, default: false
 
+    desc "Maximum number of threads to run the application with. Each will spawn its own consumer"
+    integer :max_concurrency, default: 1
```

Comment:
This should be changed to processes. Also, I think we should call this _parallel_ rather than _concurrent_ now that we're not using threads. Alternatively, we could perhaps look at what the configs are called for Unicorn and Resque?

Reply:
Perhaps "workers" is better. Also – it's not really a _max_, is it? It's the actual number of workers that will be spun up.

Reply:
Yeah true. What needs to be clear though is that the number of workers * running instances should be less than or equal to the number of partitions in the Kafka topics. Especially if people are running locally and are disappointed when throughput doesn't improve on a single partitioned topic 😄